packages/api/src/handlers/ecologist-experience/ecologist-experience-proc.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Consume Prepare output for a site
+ * @param site.dataValues
+ * @returns {any}
+ */
+export const prepareResponse = a => {
+  Object.assign((({
+    ...experience
+  }) => experience)(a), {
+    createdAt: a.createdAt.toISOString(),
+    updatedAt: a.updatedAt.toISOString()
+  })
+  a.ecologistExperience = a.experience
+  delete a.experience
+  delete a.ecologistExperience.complete
```

Comment:
There is a new way of handling the completed flags. 
See the application tags:
APIRequests.APPLICATION.tags(applicationId).add('tag')
APIRequests.APPLICATION.tags(applicationId).has('tag') // True
APIRequests.APPLICATION.tags(applicationId).remove('tag') 
APIRequests.APPLICATION.tags(applicationId).has('tag') // False

Reply:
Then in the tasklist see how the tags are used to drive the status badges

Reply:
All sorted on the tags